e2e/tests/core/02-client/client_test.go

```diff
@@ -374,14 +375,14 @@ func createMaliciousTMHeader(chainID string, blockHeight int64, trustedHeight cl
 	blockID := ibctesting.MakeBlockID(hhash, 3, tmhash.Sum([]byte(invalidHashValue)))
 	voteSet := tmtypes.NewVoteSet(chainID, blockHeight, 1, tmproto.PrecommitType, tmValSet)
 
-	commit, err := tmtypes.MakeCommit(blockID, blockHeight, 1, voteSet, signers, timestamp)
+	extCommit, err := tmtypes.MakeExtCommit(blockID, blockHeight, 1, voteSet, signers, timestamp, false)
```

Comment:
I thought about making the change, but the type return is a `ExtendedCommit`, so it makes more sense to leave as `extCommit` since we perform `extCommit.ToCommit()` which changes it from `ExtendedCommit` to `Commit`

Reply:
Sounds good to me! 👍